component/src/main/ballerina/mongodb/actions.bal

```diff
@@ -14,72 +14,57 @@
 // specific language governing permissions and limitations
 // under the License.
 
-documentation {
-    The Caller Actions for MongoDB databases.
-}
+# The Caller Actions for MongoDB databases.
 public type CallerActions object {
-    documentation {
-        The find operation implementation which selects a document in a given collection.
 
-        P{{collectionName}} The name of the collection to be queried
-        P{{queryString}} Query to use to select data
-        R{{}} `json` result from the find operation or `error` if an error occurs
-    }
+    # The find operation implementation which selects a document in a given collection.
+
+    # + collectionName - The name of the collection to be queried
+    # + queryString - Query to use to select data
+    # + return - `json` result from the find operation or `error` if an error occurs
     public extern function find(string collectionName, json? queryString) returns (json|error);
 
-    documentation {
-        The findOne operation implementation which selects the first document match with the query.
+    # The findOne operation implementation which selects the first document match with the query.
 
-        P{{collectionName}} The name of the collection to be queried
-        P{{queryString}} Query to use to select data
-        R{{}} `json` The result from the findOne operation or `error` if an error occurs
-    }
+    # + collectionName - The name of the collection to be queried
+    # + queryString - Query to use to select data
+    # + return - `json` The result from the findOne operation or `error` if an error occurs
     public extern function findOne(string collectionName, json? queryString) returns (json|error);
 
-    documentation {
-        The insert operation implementation which inserts a document to a collection.
+    # The insert operation implementation which inserts a document to a collection.
 
-        P{{collectionName}} The name of the collection
-        P{{document}} The document to be inserted
-        R{{}} `nil` or `error` if an error occurs
-    }
+    # + collectionName - The name of the collection
+    # + document - The document to be inserted
+    # + return - `nil` or `error` if an error occurs
     public extern function insert(string collectionName, json document)
         returns (error?);
 
-    documentation {
-        The delete operation implementation which deletes documents that match the given filter.
+    # The delete operation implementation which deletes documents that match the given filter.
 
-        P{{collectionName}} The name of the collection
-        P{{filter}} The criteria used to delete the documents
-        P{{multi}} Specifies whether to delete multiple documents or not
-        R{{}} `int` deleted count or `error` if an error occurs
-    }
+    # + collectionName - The name of the collection
+    # + filter - The criteria used to delete the documents
```

Comment:
The criteria `to be` used to delete the documents

Reply:
Fixed in 8a667b927e85d5481090f39f476d26c921626a4f